modules/etcd-certificates.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * authentication/understanding-certificates.adoc
+
+[id="etcd-certificates_{context}"]
+= etcd certificates
```

Comment:
Should we talk about the CA's and the validity of these? How long they are valid for; how to rotate them, etc? 
@hexfusion can you comment on this?

Reply:
@hexfusion Can you provide that information?